backend/hasura-actions/src/Query.res

```diff
@@ -10,18 +10,18 @@ module CreatePaymentStream = %graphql(`
 `)
 
 module GetStreamData = %graphql(`
-  query {
-    streams(where: {state: {_eq: "OPEN"}}) {
+  query GetStreamData ($currentTimestamp: Int!){
+    streams(where: {state: {_eq: "OPEN"}, nextPayment: {_lte: $currentTimestamp}}){
```

Comment:
```suggestion
    streams(where: {state: {_eq: "OPEN"}, nextPayment: {_lte: $currentTimestamp}, startPayment: {_gte: $currentTimestamp}}}){
```
So it doesn't pick up payments that start in the future?

Reply:
Correct, it only picks up the payment streams that need to be processed in that tick.